src/sharedData/components/SharedDataEntryBase.js

```diff
@@ -233,7 +242,7 @@ const SharedDataEntryBase = props => {
 
         <Grid
           item
-          xs={4}
+          xs={5}
```

Comment:
The description no longer lines up with the start of the buttons -- might be worth confirming with Hana if this is ok